src/telemetry.ts

```diff
@@ -28,7 +28,7 @@ export function activate(): void {
 
 export function deactivate(): void {
     if (telemetryReporter) {
-        telemetryReporter.dispose();
+        telemetryReporter.dispose().then(() => {}, () => {});
```

Comment:
Please revert.  This looks like something we may want to address in the future, but it wouldn't be part of this PR.